packages/rocketchat-custom-sounds/client/admin/adminSounds.js

```diff
@@ -1,4 +1,5 @@
-/* globals RocketChatTabBar */
+import { RocketChatTabBar } from 'meteor/rocketchat:lib';
```

Comment:
@graywolf336 well, it seems you're on a globals-killing-spree already ;)

Reply:
Heh, figured I would start the process ;)